Requirements review (#27)
elisa-tech/wg-medical-devices: applying-stpa/l2_reqs.txt

REQ-9: OpenAPS shall check whether the software update is effective and conforming to OpenAPS recommendatoins in the context of the versions of the other components in the users system.
REQ-10: SEE REQ-9 (OpenAPS shall require specific versions of dependencies which have been tested to be compatible with all of the components.)
REQ-11: Human shall be warned in documentation not to install too many packages.

> REQ-12: There shall be continuous testing of OpenAPS dependencies with when the OS is updated by the developers, to ensure new set of dependencies are compatible with updated OS.

Reviewer: What about at runtime?

> REQ-12: There shall be continuous testing of OpenAPS dependencies with when the OS is updated by the developers, to ensure new set of dependencies are compatible with updated OS.

Reviewer: ```suggestion
REQ-12: There shall be continuous testing of OpenAPS dependencies when the OS is updated by the developers, to ensure the OpenAPS dependencies on can be met by updated OS.
```

REQ-42: Device must be kept from interference by many other signals and if the network connection falters the insulin pump shall stop using OpenAPS and fallback to defaults.
REQ-43: RasPi OS must believe that it is looping when it is looping.
REQ-44: RasPi OS must be able to execute all the commands that OpenAPS app sends to RasPi OS.

> REQ-45: Fallback.

Reviewer: Please replace with REQ-45: DELETED was (Fallback).

Reviewer: Thanks!
